portal-web/test/functional/com/liferay/portalweb/tests/development/tools/testinginfrastructure/block/macro/TestUtils.macro

```diff
@@ -1,5 +1,19 @@
 definition {
 
+	@summary = "Default summary"
+	macro assertContain(actual = null, expected = null) {
+		echo(${expected});
```

Comment:
we have 'TestUtils.assertEquals' I think we don't need to add a new method

Reply:
every object has its own ERC, so I think maybe we can get it by 'var actual = JSONUtil.getWithJSONPath(${responseToParse}, "$.items[?(@.externalReferenceCode=="testErc")]['name']");'

Reply:
I generated part of JSON/JSONL with random externalReferenceCode, so the above solution doesn't suit for each case.